snapcraft/cli/remote.py

```diff
@@ -101,6 +101,9 @@ def remote_build(
         snapcraft remote-build --recover 47860738
         snapcraft remote-build --status 47860738
     """
+    echo.warning(
+        "snapcraft remote-build does not have a stable CLI interface. Use with caution in scripts."
```

Comment:
When I said similar I was thinking about mentioning upcoming workflow changes, like the authentication mechanism and ssh keys.

Reply:
Oh ok rewording that.